package.json

```diff
@@ -128,5 +127,8 @@
     "webpack": "^4.46.0",
     "webpack-cli": "^3.3.12",
     "webpack-shell-plugin": "^0.5.0"
+  },
+  "engines": {
+    "yarn": "YARN NO LONGER USED - use npm instead."
```

Comment:
is this a good practice, when the engine is going to be deprecated? I suppose that this field should take a version number, not a warning massage.